api/python/tests/test_data_transfer.py

```diff
@@ -371,14 +371,25 @@ def test_upload_large_file_etag_match_metadata_mismatch(self):
 
 
     def test_multipart_upload(self):
-        path = DATA_DIR / 'large_file.npy'
+        name = 'very_large_file.bin'
+        path = pathlib.Path(name)
+
+        size = 30 * 1024 * 1024
+        chunksize = 8 * 1024 * 1024
```

Comment:
Don't we want to push for the case where we have more than 10,000 parts per `Part number must be an integer between 1 and 10000, inclusive`

Reply:
Or at least where `size/chunksize > CHUNK_LIMIT`?

Reply:
Sure... I'll add it to the multipart copy test, though. The logic is the same - but a unittest creating and uploading a 100GB file seems a bit extreme. In the copy test, there's no actual IO going on, so it's fine.

Reply:
(That is, in the upload test, it wouldn't take up 100GB on disk, and it wouldn't be uploaded - but it would still be read)